src/components/Section/History/Employment/EmploymentItem.test.jsx

```diff
@@ -222,4 +222,14 @@ describe('The employment component', () => {
     expect(EmploymentItem.defaultProps.onUpdate()).toEqual(undefined)
     expect(EmploymentItem.defaultProps.onError(null, [])).toEqual([])
   })
+
+  it('hides APO for address of employment if self employed', () => {
+    const props = {
+      EmploymentActivity: { value: 'SelfEmployment' },
+    }
+
+    const component = createComponent(props)
+    const employmentAddressParent = component.find("[data-fieldName='employmentAddress']")
```

Comment:
👍 to adding a `data` attribute for test selectors! Can we call it something more specific like `data-test-id` to indicate that's what it's for?